deployment/ansible/roles/raster-foundry.nexus/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Install PyYAML
+  pip: name=pyyaml version={{ pyyaml_version }}
+
+- name: Create directory for persistant Nexus data
+  file: name=/opt/nexus-data state=directory
+        mode=755 owner=200 group=200
+        recurse=true
+
+- name: Add Nexus Nginx config
```

Comment:
For configuration files, we typically want to enqueue a service restart action so that the change can go into effect at the end of the Ansible run. Something like:

```yaml
notify:
    - Restart Nginx
```

Reply:
That’s already enqueued by the task that follows, `Enable Nexus Nginx config`

Reply:
Right, but if this scenario occurs, no restart will be triggered:

- Configuration file contents change
- Symlink for configuration file already present

In this case, I think it may need to be on both tasks to account for the distinct paths.